src/state/index.ts

```diff
@@ -16,13 +16,17 @@ import { routingApi } from './routing/slice'
 import swap from './swap/reducer'
 import transactions from './transactions/reducer'
 import user from './user/reducer'
+import wallet from './wallet/reducer'
+import walletOverride from './walletOverride/reducer'
 
-const PERSISTED_KEYS: string[] = ['user', 'transactions', 'lists']
+const PERSISTED_KEYS: string[] = ['user', 'transactions', 'lists', 'walletOverride']
 
 const store = configureStore({
   reducer: {
     application,
     user,
+    wallet,
+    walletOverride,
```

Comment:
`walletOverride` is confusingly named. What do you think about moving it to `user.wallet`?